doc/Language/filename-extensions.rakudoc

```diff
@@ -0,0 +1,49 @@
+=begin pod :kind("Language") :subkind("Language") :category("beginning")
+
+=TITLE Filename extensions
+
+=SUBTITLE The extensions recommended for files with Raku content.
+
+=begin table
+File contents      | Extensions recommended or used at one time | Current recommendation
+===================================================================================
+B<Language files>  |                                            |
+Raku script        | .pl, .p6                                   | .raku
+Raku module        | .pm, .pm6                                  | .rakumod
+Raku documentation | .pm, pm6, pod, pod6                        | .rakudoc
+Test files in raku | .t                                         | .rakutest
+NotQuitePerl or NQP |                                           | .nqp
+B<Module specific files> |                                      |
+Cro templates      |                                            | .crotmp
+
+=end table
+
+=head1 Introduction
+
+A filename extension is defined, at least for this page, as the remaining letters after and including
+the final '.'. A filename without a '.' in the filename does not have a filename extension.
+
+Filename extensions can be used to associate behaviour with the file content, such as opening
+a specialised reader or editor, running a program with the file content as input, etc. On some
+operating systems, this is almost the required behaviour. Even when it is not required by the
+operating system, filename extensions provide a useful mnenomic, or reminder, about the contents
+of the file.
+
+Early filesystems, the part of the operating system handling filenames, constrained filename extensions
+to three characters after the '.'. This is no longer a constraint.
+
+=head1 History and Extensions
```

Comment:
Maybe simply history. But if you decide to keep "and extensions", just put it in lowercase.